### PR TITLE
Modifier for altgr

### PR DESCRIPTION
**Pull request by elferherrera**
Added clause to consider special modifier for non english keyboards

### CODE REVIEW

File: src/edit_mode/emacs.rs [hunk 1]
```diff
@@ -33,6 +33,9 @@ impl EditMode for Emacs {
                 | (KeyModifiers::SHIFT, KeyCode::Char(c)) => {
                     ReedlineEvent::EditInsert(EditCommand::InsertChar(c))
                 }
+                (m, KeyCode::Char(c)) if m == KeyModifiers::CONTROL | KeyModifiers::ALT => {
```
Comment (jntrnr) [inline src/edit_mode/emacs.rs:36]: Since we look for Ctrl+c a few lines above this one, should we just look for ALT here?

**Comment by jntrnr** [comment]
Just fixed CI for reedline, so in theory the next change should pick it up

File: src/edit_mode/emacs.rs (hunk 1 above)
Comment (elferherrera) [reply src/edit_mode/emacs.rs:36]: The thing is that in my keyboard the key `alt gr` appear as a mixed KeyModifier called `CONTROL | ALT`. This isnt a Control or Alt situation, it has to be both

Comment (jntrnr) [reply src/edit_mode/emacs.rs:36]: Oooh as a bitwise operation. Got it. I totally misread that this morning.

**Pull request merged**